packages/core/src/validate.js

```diff
@@ -5,6 +5,7 @@ import { deepEquals, getDefaultFormState } from "./utils";
 
 let formerCustomFormats = null;
 let formerMetaSchema = null;
+const ROOT_SCHEMA_PREFIX = "__rjsf_rootSchema";
```

Comment:
Let's add a trailing `_`, just so that the generated schema names will be more readable

```suggestion
const ROOT_SCHEMA_PREFIX = "__rjsf_rootSchema_";
```

Reply:
Never mind, we'll have a `/` at the end anyway.